src/modules/localization/src/frame_tf_calibrate/frame_calibration.cpp

```diff
@@ -0,0 +1,231 @@
+#include "localization/frame_tf_calibrate/frame_calibration.h"
+/*
+NOTE: This node listens to the tf broadcasts and performs the homogeneous transforms.
+      Another major function of this node is to "calibrate" the initial position and orientation
+      of the vehicle. The homogeneous transforms are applied and then the result is published.
+      
+INPUTS: TOPIC:  /gps/geodesy_odom
+                    Msg: nav_msgs::Odometry
+        TOPIC:  /localization/rotated_yaw

```

Comment:
I see a lot of ```/localization/rotated_yaw```, but I don't see the file that contains the publishing node.
Also, if the topic only publishes a rotated yaw value, why not include it in the frame_calibration file as well?

Reply:
That was going to come up when I started cleaning things up, but I'll just include that change. It's in the imu_adapter.cpp file just for reference.

Reply:
We should take out the `tf_master.cpp` in the `geodesy` module as well and remove the logic for that in the `control` module as well in another issue.

Reply:
Raise that issue once this PR is merged. Tag @drofp and I for it and include a small description of what needs to be changed so we remove those part of pseudo-localization code from our modules.

Reply:
Sounds good. I will delete `imu_tf_adapter` and copy-paste its contents into the `master_tf` module.

Reply:
#156